Classes/Domain/Model/Book.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Books\Domain\Model;
+
+final class Book
+{
+    /** @var string */
+    private $title;
+
+    /** @var int */
+    private $yearOfPublication;
+
+    public function __construct()
+    {
+    }
+
+    /**
+     * @return string
```

Comment:
I'd prefer to drop the PHPDoc for cases where if does not add anything to the existing type declarations. (for all methods)